papyrus/geo_interface.py

```diff
@@ -40,6 +40,15 @@ class Spot(Base):
             __tablename__ = 'spots'
             id = Column(Integer, primary_key=True)
             geom = Geometry('the_geom', Geometry('POINT', 4326))
+
+    Properties can be flagged read only through column info properties.
+    Such column will not be updated without error.
```

Comment:
I don't see errors being triggered when attempting to edit readonly properties.

Reply:
The class doc was saying without error, but I feel better to raise an exception.
I changed this behavior to get a HTTPBadRequest here but I feel wreid to depend on pyramid in GeoInterface. Tell me if you prefer a custom exception here.

Reply:
Sorry, I had misread the doc strings.